app/validation/validators.py

```diff
@@ -66,7 +66,7 @@ def __init__(self, min=None, max=None, messages=None):
         self.messages = messages
 
     def __call__(self, form, field):
-        data = field.data
+        data = int(field.data)
```

Comment:
I don't think there should be a need to cast this as NumberRanges should only be applied to IntegerFields.